products/compute/api.yaml

```diff
@@ -11452,32 +11527,742 @@ objects:
+                    backendService.
+                  item_type: !ruby/object:Api::Type::NestedObject
+                    properties:
+                      - !ruby/object:Api::Type::String
+                        name: 'headerName'
+                        description: |
+                          The name of the header.
+                      - !ruby/object:Api::Type::String
+                        name: 'headerValue'
+                        description: |
+                          The value of the header to add.
+                      - !ruby/object:Api::Type::Boolean
+                        name: 'replace'
+                        default_value: false
+                        description: |
+                          If false, headerValue is appended to any values that already exist for the
+                          header. If true, headerValue is set for the header, discarding any values that
+                          were set for that header. Defaults to false.
+                - !ruby/object:Api::Type::Array
+                  name: 'requestHeadersToRemove'
+                  item_type: Api::Type::String
+                  description: |
+                    A list of header names for headers that need to be removed from the request
+                    prior to forwarding the request to the backendService.
+                - !ruby/object:Api::Type::Array
+                  name: 'responseHeadersToAdd'
+                  description: |
+                    Headers to add the response prior to sending the response back to the client.
+                  item_type: !ruby/object:Api::Type::NestedObject
+                    properties:
+                      - !ruby/object:Api::Type::String
+                        name: 'headerName'
+                        description: |
+                          The name of the header.
+                      - !ruby/object:Api::Type::String
+                        name: 'headerValue'
+                        description: |
+                          The value of the header to add.
+                      - !ruby/object:Api::Type::Boolean
+                        name: 'replace'
+                        default_value: false
+                        description: |
+                          If false, headerValue is appended to any values that already exist for the
+                          header. If true, headerValue is set for the header, discarding any values that
+                          were set for that header. Defaults to false.
+                - !ruby/object:Api::Type::Array
+                  name: 'responseHeadersToRemove'
+                  item_type: Api::Type::String
+                  description: |
+                    A list of header names for headers that need to be removed from the response
+                    prior to sending the response back to the client.
+            - !ruby/object:Api::Type::String
+              name: 'name'
```

Comment:
Can you break out the refactorings on future changes? It's hard to tell that this was an old field, and looks like we're adding a `Required` field.

Reply:
Will do.